config/settings/development.yml

```diff
@@ -12,5 +12,5 @@ betamocks:
   recording: false
   # the cache dir depends on how you run the api, run `bin/spring stop` after switching this setting
   #cache_dir: ../vets-api-mockdata # via rails; e.g. bundle exec rails s or bundle exec rails c
-  cache_dir: /cache # via docker; e.g. make up or make console
```

Comment:
may want to avoid committing this change - it should be possible to apply this setting in `config/settings.local.yml`